tests/db_functions/test_replace.py

```diff
@@ -0,0 +1,56 @@
+from django.db.models import F, Value
+from django.db.models.functions import Concat, Replace
+from django.test import TestCase
+
+from .models import Author
+
+
+class ReplaceTests(TestCase):
+
+    @classmethod
+    def setUpTestData(cls):
+        Author.objects.create(name='George. R. R. Martin')
```

Comment:
`George. R. R. Martin` → `George R. R. Martin` (Remove the extra period, and throughout below.)

Reply:
OK. This was just copy pasted from the `StrIndex` function.